src/app/app.component.ts

```diff
@@ -45,9 +48,9 @@ export class AppComponent extends GraphExplorerComponent implements OnInit, Afte
         ClientId: '',
         Language: 'en-US',
         DefaultUserScopes: 'openid profile User.ReadWrite User.ReadBasic.All Sites.ReadWrite.All Contacts.ReadWrite ' +
-            'People.Read Notes.ReadWrite.All Tasks.ReadWrite  Mail.ReadWrite Files.ReadWrite.All Calendars.ReadWrite',
+            'People.Read Notes.ReadWrite.All Tasks.ReadWrite Mail.ReadWrite Files.ReadWrite.All Calendars.ReadWrite',
         AuthUrl: 'https://login.microsoftonline.com',
-        GraphUrl: getParameterByName('GraphUrl') || 'https://graph.microsoft.com',
+        GraphUrl: getParameterByName('GraphUrl') || getGraphUrl(),
```

Comment:
The URL is not set to the canary endpoint when I access GE with the query param http://localhost:3000/?mode=canary when a user is not signed in. Set the canary URL when accessing GE without a sign-in user.

Reply:
We get an extra '/' in the URL https://canary.graph.microsoft.com//v1.0/me/.

Reply:
Yes. Sounds better to change the string.